### PR TITLE
Measurements sidecar

### PR DESCRIPTION
**Pull request by joverlee521**
Adds "measurements" as a subresource type & sidecar file.
I searched for "tip-frequencies" within the repo and added "measurements" to the appropriate places.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -291,6 +291,7 @@ const schemaRoutes = [
   ["/schemas/dataset/v2",              "https://github.com/nextstrain/augur/raw/master/augur/data/schema-export-v2.json"],
   ["/schemas/dataset/root-sequence",   "https://github.com/nextstrain/augur/raw/master/augur/data/schema-export-root-sequence.json"],
   ["/schemas/dataset/tip-frequencies", "https://github.com/nextstrain/augur/raw/master/augur/data/schema-tip-frequencies.json"],
+  ["/schemas/dataset/measurements",    "https://github.com/nextstrain/augur/raw/master/augur/data/schema-measurements.json"],
```
Comment (jameshadfield) [inline src/app.js:294]: This github URL doesn't exist (yet). There's https://github.com/nextstrain/augur/pull/827 but it's not yet ready for merge, and it doesn't name the file `schema-measurements.json` either.

**Review by jameshadfield — COMMENTED** [review]
Tested the charon endpoints via http://localhost:5000/charon/getDataset?prefix=community/joverlee521/nextstrain-testing/flu/seasonal/h1n1pdm/ha/09-17&type=measurements ✅

File: src/app.js (hunk 1 above)
Comment (jameshadfield) [reply src/app.js:294]: Ahhh, I now see your commit comment covers this: "Currently gets redirected to a 404 page, but will soon be added as a JSON schema in augur."

**Comment by joverlee521** [comment]
Rebased onto the auspice release, so now the [testing app can displays the measurements panel](https://nextstrain-s-measuremen-tlovqh.herokuapp.com/community/joverlee521/nextstrain-testing/flu/seasonal/h1n1pdm/ha/09-17).

**Review by tsibley — APPROVED** [review]
:+1: FWIW, the first three commits feel like they should be lumped together to me since they're all dealing with the same thing: accounting for a new sidecar type by adding the "measurements" key places. But not a big deal at all and certainly not a must.

Would you also update our [data formats page](https://docs.nextstrain.org/en/latest/reference/data-formats.html)?

**Pull request merged**